.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: groovy
+os: linux
+dist: bionic
+arch: amd64
+sudo: required
+services: docker
+
+before_install:
+    - echo "This is the first attempt at .travis.yml for variantcalling"
+    - echo "This is an attempt to build the Dockerfile environment"
+    - docker build . --file ./Dockerfile --tag variantcaller
```

Comment:
Please move building of Docker image to Docker Hub and pull the image from Docker Hub for CI.